lib/matplotlib/tests/test_axes.py

```diff
@@ -2454,6 +2454,17 @@ def test_errorbar_limits():
     ax.set_title('Errorbar upper and lower limits')
 
 
+def test_errobar_nonefmt():
+    # Check that passing 'none' as a format still plots errorbars
+    x = np.arange(5)
+    y = np.arange(5)
+
+    plotline, _, barlines = plt.errorbar(x, y, xerr=1, yerr=1, fmt='none')
+    assert plotline is None
+    for errbar in barlines:
+        assert np.any(errbar.get_color() != [0, 0, 0, 0])
```

Comment:
I'm not sure if this is the best way to check that the errorbars aren't see-through, so other suggestions welcome.